__tests__/api.test.js

```diff
@@ -0,0 +1,85 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const { handleServerErrors } = require('../controllers/error-handling-controller');
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+
+describe('GET /api/topics', () => {
+    test('responds with a 200 status code', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
+    test('get /api/topics request responds with an array of topics objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(res => {
+            expect(res.body.topics).toBeInstanceOf(Array);
+            expect(res.body.topics).not.toBeEmpty();
+            expect(res.body.topics[0]).toHaveProperty('slug');
+            expect(res.body.topics[0]).toHaveProperty('description');
```

Comment:
These two lines are unnecessary because you're checking for these properties in the for loop below